src/react-apps/applications/service-development/src/sharedResources/appCluster/appClusterReducer.ts

```diff
@@ -0,0 +1,71 @@
+import update from 'immutability-helper';
+import { Action, Reducer } from 'redux';
+import * as AppClusterActionTypes from './appClusterActionTypes';
+import * as AppClusterActions from './getDeployments/getDeploymentsInAppClusterActions';
+
+export interface IAppClusterState {
+  deploymentList: any;
+}
+
+const initialState: IAppClusterState = {
+  deploymentList: {
+    at21: {
```

Comment:
This will make it weird to loop through. Could we have an array besides this with definitions for all the environments? Like we did with formLayoutOrder and formLayout (before the changes we did with the rewriting of formLayout).

Might be something we should do in the rest of #2713

Reply:
Yes, we have to fix this somehow during #2713. I've only moved the resource in this PR.